HelgaShiryaeva/assignment1/ReversedListAccessUtil

```diff
@@ -0,0 +1,91 @@
+public class ReversedListAccessUtil {
+    public <T> T getReversed(int index, MyList<T> list) {
```

Comment:
Name of the method does not really say what it does. Would some other name be better?

Also please add comment to provide more details.